src/bot.py

```diff
@@ -0,0 +1,23 @@
+import discord
+from discord.ext.commands import Bot
+from settings import get_settings
+from commands.exile import create_exile_command, create_unexile_command
+import logging
+
+settings = get_settings()
+logger = logging.getLogger(__name__)
+
+
+class ModdingwayBot(Bot):
+    async def setup_hook(self):
+        logger.debug("setting up commands")
+        create_unexile_command(self)
+        create_exile_command(self)
```

Comment:
i think we could shove this in a helper function so that we don't have to revisit this function every time we add a new command

Reply:
I pulled this into a method outside the setup hooks event, and combined the two exile command